src/ansible_navigator/actions/_actions.py

```diff
@@ -111,17 +159,32 @@ def run_interactive(package: str, action: str, *args: Any, **_kwargs: Any) -> An
 
 
 def run_interactive_factory(package: str) -> Callable:
-    """Create a call() function for one package"""
+    """Create a ``run_interactive`` function for one package.
+
+    :param package: The name of the package
+    :returns: A partial ``run_interactive`` method for the package
+    """
     return functools.partial(run_interactive, package)
 
 
 def run_stdout(package: str, action: str, *args: Any, **_kwargs: Any) -> Any:
-    """Call the given action's run_stdout"""
+    r"""Call the given action's ``run_stdout`` method.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :param \*args: The arguments passeed to the action's run_stdout method
+    :param \**_kwargs: The keyword arguments passed to the action's run_stdout method
+    :returns: The outcome of running the actions' run_stdout method
+    """  # noqa: DAR101, DAR102 # *args, **kwargs escaped for sphinx
     action_cls = get(package, action)
     args = args[0]
     return action_cls(args).run_stdout()
 
 
 def run_stdout_factory(package: str) -> Callable:
-    """Create a run_stdout() function for one package"""
+    """Create a ``run_stdout`` function for one package.
```

Comment:
I'd like for such references to keep `()` because this is what Sphinx appends to normal callable-like roles when they are rendered:
```suggestion
    """Create a ``run_stdout()`` function for one package.
```

Reply:
I _thought?_ this caused a sphinx error, we'll know soon enough

Reply:
No, Sphinx doesn't care about things inside _double_ backticks — for it, it's just something that is arbitrary inline code thingy, there's no semantics inside.